cloud/helm/aws/charts/backup-restore/templates/_scripts.tpl

```diff
@@ -0,0 +1,45 @@
+{{- define "liferayAwsBackupRestore.script.getDependenciesModuleOutputs" -}}
+{{- $terraformOutputIsRestoring := "is_restoring" -}}
+{{- $terraformOutputDbRestoreSnapshotIdentifier := "db_restore_snapshot_identifier" -}}
+{{- include "liferayAwsBackupRestore.script.partial.shStart" . }}
+
+function main {
+    local db_restore_snapshot_identifier
+
+    terraform init -input=false
+
+    terraform plan -detailed-exitcode -input=false
+
+    db_restore_snapshot_identifier=$( \
+        terraform \
+            output \
+            -raw \
+            {{ $terraformOutputDbRestoreSnapshotIdentifier }} 2>/dev/null || echo "")
```

Comment:
Terraform outputs that are null throw an error, so catch it .

Not relevant for the other output since it's a boolean.